### PR TITLE
mini team meeting notes

### PR DESCRIPTION
**Pull request by edouarde1**

### CODE REVIEW

File: docs/meetings/team/2023_01_16.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+- Frame-by-frame Solution:
+  - Create a frame for every mouth and animation modification
+  - This would be a lot of work
+
+- Exploration Task:
+  - TASK: Explore different solutions to add realism into our animations.
+    - A solution that is not too much manual work and also not too complex
+    - SVG.js
+      - Pretty sure @matt looked at this library, what was the consenus on it ?
```
Comment (LoganParker) [inline docs/meetings/team/2023_01_16.md:43]: We did not like it I believe

**Review by LoganParker — COMMENTED** [review]
Thank you!! Sorry for missing this one

**Review by LoganParker — APPROVED** [review]

**Comment by LoganParker** [comment]
👑

File: docs/meetings/team/2023_01_16.md (hunk 1 above)
Comment (edouarde1) [reply docs/meetings/team/2023_01_16.md:43]: no

**Pull request merged**